server/models/handlers/workerTimeslot.js

```diff
@@ -10,6 +10,17 @@ async function addWorkerTimeslot(slotId, schoolId, workerId, status, date) {
         return false;
     }
 };
+async function updateIndividualWorkerAvailability(workerTimeslotId, newStatus) {
```

Comment:
Hi Peter,
I am not sure where this method is getting called. On line 67 of the `handlers/appointment.js` file, it calls the workerTimeslotModel rather than the handler. Thus, is this method necessary?

Reply:
Amy this is a great catch, I will remove this